src/guides/workflows_references.md

```diff
@@ -0,0 +1,9 @@
+# Passing Data Between Nodes
+
+To use the output of a workflow node as the input of a connected downstream node, click on the <code><Icon icon="fas fa-link" /></code> button in a node's editor and select the data alias from the drop-down menu:
+
+<img alt="Selecting a reference." src="/_images/workflows_reference.png" center>
+
+::: tip
+[View the data compatibility reference.](/reference/workflow_data_types.md)
```

Comment:
This tip can be clearer. If a user sees this, I don't expect them to know what "data compatibility reference" means.

The label also doesn't match the title of the linked article.

I suggest something more explicit like:

"To understand which input and output types are exact matches, compatible, or incompatible, see the [Workflow Node Data Types] reference."